ChangeLog.md

```diff
@@ -1,3 +1,9 @@
+# Changes in 1.0.3
+
+* Fix an inconsistency in `clap_plugin->destroy()` documentation:
+  it is now **required** to deactivate the plugin before destroying it.
+* Improve documentation for `clap_host_params->request_flush()`.
+
```

Comment:
+ improve documentation for entry.h should be here also

Reply:
Done